### DISCUSSION TITLE
Images as External Links

### DISCUSSION THREAD

**Discussion by sophia-kobelja** [directus/directus#10889] [Archive]
Can you make it easy for an image to be used as an external link? E.g., when inserting an image into blog content, provide an additional field for "External Link", which will wrap the image in an `a` tag with the href to the provided link? 

Providing screenshot here for clarity on where this feature would be useful:
![image](https://user-images.githubusercontent.com/91899004/148413349-9d203c44-e306-4b33-91e8-a5fc35c1fbaa.png)

**Comment by rijkvanzanten** [comment]
@sophia-kobelja I believe you can already achieve that by selecting the image in the wysiwyg itself, and clicking on the insert-link button 🤔

**Reply by sophia-kobelja** [reply]
Oh wow. Thank you!
